aws/rust-runtime/aws-inlineable/Cargo.toml

```diff
@@ -29,7 +29,7 @@ http-body = "0.4.5"
 md-5 = "0.10.1"
 ring = "0.16"
 tokio = { version = "1", features = ["full"] }
-tokio-stream = "0.1.7"
+tokio-stream = "0.1.5"
```

Comment:
intentional downgrade?

Reply:
Yeah, I picked the minimum version across all the `tokio-stream` references.